gui/dig.lua

```diff
@@ -5,18 +5,22 @@
 
 -- Must Haves
 -----------------------------
+-- Better UI, it's starting to get really crowded
```

Comment:
tap TaxiService for the design. I can point you to examples for the implementation.